src/WCSimDetectorConstruction.cc

```diff
@@ -120,13 +120,9 @@ WCSimDetectorConstruction::WCSimDetectorConstruction(G4int DetConfig,WCSimTuning
 
   messenger = new WCSimDetectorMessenger(this);
 
+
   // Get WCSIMDIR
-  const char *wcsimdirenv = std::getenv("WCSIMDIR");
```

Comment:
why is this removed?

Reply:
My mistake from other version of WCSim. It's solved now.